tox.ini

```diff
@@ -90,11 +90,13 @@ commands =
     notudf: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} and not udf" {posargs:} src/snowflake/snowpark tests
     udf: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} or udf" {posargs:} src/snowflake/snowpark tests
     notdoctest: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} or udf" {posargs:} tests
+    multithreaded: {env:SNOWFLAKE_PYTEST_CMD} --multithreading_mode -m "{env:SNOWFLAKE_TEST_TYPE} and multithreaded_run" {posargs:} tests
```

Comment:
so we are running all tests, but some tests are written in multithreaded way?

Reply:
tests that are tagged with `@multithreaded_run()` with run in multi-threaded way. Others will run regularly

Reply:
that means we will have a lot of duplicated tests with other daily job, note that all our jobs are run on the same cluster, running to many tests could make our merge gate slow. One thing we can do is to use markers, and only run the tests with the marker, you can take a look at the maker "@pytest.mark.modin_sp_precommit", which is used to control the tests that are run in the precommit job.